### DISCUSSION TITLE
Wrapped tokens?

### DISCUSSION THREAD

**Discussion by an0ndroid** [crypto-org-chain/chain-main#498] [Ideas] [answered]
Any soft ETA on when we can start launching new tokens based on the crypto.org mainnet? The CRO community is exited to build value into the system.

**Comment by tomtau** [comment, marked as answer]
If this proposal passes: https://explorer.nebkas.ro/proposals/2 then from May 1, the IBC fungible token transfer application is enabled.
This enables transferring in new tokens as defined in https://github.com/cosmos/ibc/tree/master/spec/app/ics-020-fungible-token-transfer

Note that there appears to be a bug for the "solo machine" client: https://github.com/cosmos/ibc/issues/562 which may be a temporarily blocker for applications that intended to use the solo machine client.
